src/cpu-template-helper/src/utils/mod.rs

```diff
@@ -64,6 +66,40 @@ pub enum UtilsError {
     CreateVmResources(vmm::resources::ResourcesError),
     /// Failed to build microVM: {0}
     BuildMicroVm(#[from] StartMicrovmError),
+    /// Failed to create temporary file: {0}
+    CreateTempFile(#[from] vmm_sys_util::errno::Error),
+    /// Failed to operate file: {0}
+    FileIo(#[from] std::io::Error),
+}
+
+// Utility function to prepare scratch kernel image and rootfs and build mock Firecracker config.
+fn build_mock_config() -> Result<(TempFile, TempFile, String), UtilsError> {
+    let kernel = TempFile::new()?;
+    kernel.as_file().write_all(
+        #[cfg(target_arch = "aarch64")]
+        include_bytes!("mock_kernel/aarch64.bin"),
+        #[cfg(target_arch = "x86_64")]
+        include_bytes!("mock_kernel/x86_64.bin"),
+    )?;
```

Comment:
Nice trick!

Reply:
Yeees! I run into this feature by chance in another place :)